lighthouse-core/test/index-test.js

```diff
@@ -103,8 +102,7 @@ describe('Module Tests', function() {
     try {
       await lighthouse('file:///a/fake/index.html', {}, {});
     } catch (err) {
-      expect(err.friendlyMessage)
-          .toBeDisplayString('The URL you have provided appears to be invalid.');
+      expect(err.friendlyMessage).toBe('The URL you have provided appears to be invalid.');
```

Comment:
runner [localizes these internally](https://github.com/GoogleChrome/lighthouse/blob/d236a70daf476f2ffa09b30853cb60b3caac79e0/lighthouse-core/runner.js#L177-L180) (and [there's a test for that](https://github.com/GoogleChrome/lighthouse/blob/d236a70daf476f2ffa09b30853cb60b3caac79e0/lighthouse-core/test/runner-test.js#L828-L847)), so they aren't `LH.IcuMessage`s by the time `index.js` sees them